### PR TITLE
update modules and fix #176

### LINKED ISSUE #176: clipboard.js and RequireJS
I've been racking my brain on it, but I can't figure out the right way to wrap `clipboard.js` to work with RequireJS. Has anybody pulled that off before?

### PR DESCRIPTION
**Pull request by heldr**
Notes:
* Babel 5 is deprecated and Babel 6 has lot of concept changes
* PhantomJS 2 finally :pray: includes **.bind**, it no longer needs polyfill
* Babel plugin to transpile modules to UMD in order to get AMD compatibility #176 
* The second commit fixes a [Babel 6 change](http://www.2ality.com/2015/12/babel-commonjs.html#why_isnt_the_default_export_done_like_a_commonjs_single_export%3F) on ``export default`` behaviour.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,29 +16,34 @@
...
+    "karma-browserify": "^5.0.1",
     "karma-chai": "^0.1.0",
     "karma-mocha": "^0.2.0",
-    "karma-phantomjs-launcher": "^0.2.1",
+    "karma-phantomjs-launcher": "^1.0.0",
     "karma-sinon": "^1.0.4",
     "mocha": "^2.3.3",
-    "phantomjs": "^1.9.18",
-    "phantomjs-polyfill": "0.0.1",
+    "npm": "^3.7.3",
```
Comment (zenorocha) [inline package.json]: Why would we need npm as a dependency? They will not be able to run the script without it anyways.

Comment (heldr) [inline package.json]: npm did it, gonna remove LOL

**Comment by zenorocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by zenorocha** [comment]
Awesome!!! Thanks @heldr ;)

**Comment by heldr** [comment]
NP! :tea:

**Pull request merged**